packages/client/package.json

```diff
@@ -71,16 +71,15 @@
   "dependencies": {
     "@adaptive/hydra-platform": "4.57.0",
     "@finos/fdc3": "^1.2.0",
-    "@openfin/core": "31.75.4",
-    "@openfin/workspace": "13.0.7",
-    "@openfin/workspace-platform": "13.0.7",
+    "@openfin/core": "33.77.11",
```

Comment:
not much of a bump in the runtime, but should be enough to cure some of the long-standing OF bugs that we had.
going to latest WS/Runtime was problematic when running up locally on Mac - EPERM error on second run once the runtime has been downloaded .. WS 16 / Runtime 33 does not suffer from this. Subject of (yet) another OF support issue soon ....
At least this / explicit register() fixes the dual Notification Services .. and the out-of-control errors when shutting down!